src/main/java/com/google/sps/servlets/CalendarTokenServlet.java

```diff
@@ -13,6 +18,25 @@
 
 @WebServlet("/token")
 public class CalendarTokenServlet extends AbstractAppEngineAuthorizationCodeServlet {
+  protected AuthorizationCodeFlow flow;
+  protected UserService userService;
+
+  @Inject
+  CalendarTokenServlet(AuthorizationCodeFlow flow, UserService userService) {
+    this.flow = flow;
+    this.userService = userService;
+  }
+
+  public CalendarTokenServlet() throws IOException {
+    this.userService = UserServiceFactory.getUserService();
+    try {
+      this.flow = Utils.newFlow();
+    } catch (GeneralSecurityException e) {
+      this.flow = null;
+      e.printStackTrace();
+    }
+  }
```

Comment:
Similar as for the AuthServlet. You need to specify another method in the module, e.g.
```
@Provides
AuthorizationCodeFlow provide AuthorizationCodeFlow() { return Utils.newFlow(); }
```
Then both UserService and the flow gets automatically injected in the constructor above and you don't need this default constructor.